servers/zts/src/test/java/com/yahoo/athenz/zts/notification/CertFailedRefreshNotificationTaskTest.java

```diff
@@ -414,6 +416,107 @@ public void testNoValidServices() {
         System.clearProperty(ZTS_PROP_NOTIFICATION_CERT_FAIL_IGNORED_SERVICES_LIST);
     }
 
+    @Test
+    public void testSomeRecordsSnoozed() {
+
+        Date currentDate = new Date();
+        List<X509CertRecord> records = new ArrayList<>();
+        System.setProperty(ZTS_PROP_NOTIFICATION_CERT_FAIL_PROVIDER_LIST, "provider");
+
+        for (int i = 0; i < 8; ++i) {
+            X509CertRecord record = getMockX509CertRecord(currentDate, i);
+            records.add(record);
+            NotificationTestsCommon.mockDomainData(i, dataStore);
+            DomainData domainData = new DomainData();
+            // Make even domains snooze
+            if (i % 2 == 0) {
+                Map<String, StringList> tags = new HashMap<>();
+                tags.put("DisableCertRefreshNotification", new StringList().setList(Arrays.asList("true")));
```

Comment:
pls use the constant value..

Reply:
I prefer not to make the const public to make it available in test as this const is only relevant for this class.
Also, if the value used in the class will change in the future (which might break domains that used the old value) I prefer to know about it by having this test fail